code/modules/admin/topic.dm

```diff
@@ -388,6 +388,28 @@
 			SSticker.real_reboot_time = 0 // If they set this at round end, show the "Reboot was cancelled by an admin" message instantly
 		href_list["secretsadmin"] = "check_antagonist"
 
+	else if(href_list["change_vip_target"])
+		if(!check_rights(R_ADMIN))
+			return
+		var/list/possible_targets = list()
+		for(var/datum/mind/possible_target in SSticker.minds)
+			if((possible_target != src) && ishuman(possible_target.current))
+				possible_targets += possible_target.current
+
+		possible_targets = sortAtom(possible_targets)
+		var/mob/living/carbon/human/new_target = input("Select new VIP:", "Objective target") as null|anything in possible_targets
+		if(!new_target)
+			return
```

Comment:
Since it's after an `input()`, you might want to make sure they've not been deleted or otherwise disappeared in the wait time.
```suggestion
		if(QDELETED(new_target))
			return
```